fenix/app/src/main/java/org/mozilla/fenix/home/recentbookmarks/controller/RecentBookmarksController.kt

```diff
@@ -46,15 +50,27 @@ class DefaultRecentBookmarksController(
     private val activity: HomeActivity,
     private val navController: NavController,
     private val appStore: AppStore,
+    private val selectTabUseCase: TabsUseCases.SelectTabUseCase,
+    private val tabs: List<TabSessionState> = activity.components.core.store.state.tabs,
```

Comment:
this is the only line that gives me a little pause. Could we instead just pass in `BrowserStore`, make it mandatory with no default value, and retrieve the tabs in `handleBookmarkClicked`?

Reply:
@MozillaNoah yes, that's a good spot, thanks!